lib/test/tester.js

```diff
@@ -211,6 +212,45 @@ class Tester {
     this.coverageFile = path.join(this.tempDir, 'coverage.out')
   }
 
+  buildGoTestArgs (timeoutMillis: number = 60000): Array<string> {
+    const args = ['test', '-coverprofile=' + this.coverageFile]
```

Comment:
Coverage is configurable. We should check to see if coverage is enabled? `atom.config.get('go-plus.test.runTestsWithCoverage')`

Reply:
Nothing changed as part of this PR.  The coverage toggle only affects whether we include `-cover` in the `go test` command (for running tests), NOT while running `go test -c` to compile tests (for build errors).